tests/setup_module_utils.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+if [ ! -d $1 -o ! -d $2 ] ; then
```

Comment:
IMHO the variables should be quoted in case the arguments contain whitespace. It is not likely but best practice IMHO. What do you think?